### ISSUE TITLE
[Bug Report][3.6.7] VProgressLinear: "aria-valuenow" always calculated for range [0, 100] regardless of "max" attribute

### ISSUE THREAD

**Issue by felixvictor** [vuetifyjs/vuetify#19894]
### Environment
**Vuetify Version:** 3.6.7
**Vue Version:** 3.4.27
**Browsers:** Chrome 125.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
Check the `aria-valuenow` attribute in the developer tools.

### Expected Behavior
`aria-valuenow` should respect the `max` attribute if given, i.e. should be in the range [0, max], see [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow)

> the `aria-valuenow` enables defining a current numeric value between the minimum and maximum values.

### Actual Behavior
`aria-valuenow` is always in the range [0, 100]

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNplj90KwjAMhV8l9GYKzoF4JdvA57BelJrJoH+k7RDG3t22m6B4U3pOki8nt5l5ks3VueMUkV1YG1A7JQL23AC0Uy2cK98ipDVBjAZps4rpyD4Jva9VqgiCqdb2garjTEYiNIEz0OKV9Jmzvm3+Jj78VPldkJ11f9t85UrSSxpdAI8hrvFG7SwFmIFwgAUGshqqdFKV2wES1wfY8kCXu3anfeaupMRgy8FEpcpzfwOdul97)
